metrics/share-of-voice.md

```diff
@@ -0,0 +1,56 @@
+- There is a competitor that comes to mind, whether its internal competition where you compare community SOV to PR or AR efforts or external, where you focus on the top 3 companies in a given space (ex. comparing how N projects from X, Y, or Z companies compete for conversation around Data Science).
+- Your competitors are measurable in the same way you are, meaning you have a representative list of their activities similar to your own.
+- You have a trusted and repeatable system of measurement (ideally software used by multiple teams for multiple programs).
+
+If these statements are not applicable to your situation, abandon SOV immediately. It’s not for you at this moment. Rely on anecdotes or something else.
+
+### Visualizations
+
+One example made with PowerPoint:
+
+![sov-metric](https://user-images.githubusercontent.com/1744971/71270812-1b27c580-2318-11ea-9188-0a79ec700348.png)
+
+### Tools Providing the Metric
+
+_This depends completely on what you've decide to measure._ Here are a few examples, which are far from an exhuastive list. Please add others as you find them.
+
+- [Google Search Console](https://search.google.com/) for relative search traffic.
+- [StackOverflow Data Explorer](https://data.stackexchange.com/stackoverflow/query/new) for SOV on StackExchange sites.
+- [Export Tweet](https://www.exporttweet.com/) to measure hashtag SOV.
+- [Hootsuite Analytics](https://hootsuite.com/platform/analytics) for broader social media SOV.
+- [Sysomos](https://sysomos.com/platform/analyze/) for social and press article SOV.
+
+### Data Collection Strategies
+
+The framing of what your measuring will always impact data collection. Here are a few examples of how you might frame the SOV analysis and then move to measure it. 
+
+1. **Impact around an event**. To secure funding for participation at an industry conference, you present the impact our community activities will have on the visibility of your community in a given space. Given the influence of Twitter on technology perception, you offer to compare the SOV of you and your community around the event in comparison to your top 3 competitors. To measure, you use Export Tweet to download all data around one or more keywords. You then use a regular expression to count the % of conversation your community had relative to other communities and show that you had an impact. For best results, establish a baseline by downloading historical data around this topic, which is likely to show a growth in SOV due to your work.
+2. **Value per budget**. Starting from a similar event strategy as #1, add an additional comparison. Collaborate with colleagues where you know how much they spend for "visibility" at conferences to show an additional angle to this measurement. The result will be something like: "we grew our percentage of SOV by 34% relative to our competitors, and we did so for X amount of money, which is 1/5th the cost of just hosting a booth or 1/3rd the cost of running an evening event." 
```

Comment:
These examples speak to "you". Not sure if we would improve the metric by changing that.